src/core/hle/service/time/time_manager.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019 yuzu emulator team
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <chrono>
+#include <ctime>
+
+#include "core/hle/service/time/time_manager.h"
+#include "core/settings.h"
+
+namespace Service::Time {
+
+static constexpr Clock::TimeSpanType standard_network_clock_accuracy{0x0009356907420000ULL};
```

Comment:
```suggestion
constexpr Clock::TimeSpanType standard_network_clock_accuracy{0x0009356907420000ULL};
```

File-scope const/constexpr variables internally link by default.